spec/system/support_interface/reasons_for_rejection_dashboard_spec.rb

```diff
@@ -90,72 +134,82 @@ def reject_application_without_structured_reasons(application_choice)
   end
 
   def then_i_should_see_reasons_for_rejection_course_full
-    expect(all('.govuk-heading-m')[0]).to have_content('Course full')
-    expect(all('.govuk-grid-row')[0]).to have_content('10% of all structured rejections')
-    expect(all('.govuk-grid-row')[0]).to have_content('2 total')
-    expect(all('.govuk-grid-row')[0]).to have_content('1 this month')
+    within '#course_full' do
```

Comment:
Would we want to keep an `expect` here for the percentage? It will still be displayed on the page so would make sense to cover this functionality.

Reply:
Yea really good point George 👍 . Will add it back in!